examples/with-nextjs-and-clerk-auth/src/components/ThemeSelect/ThemeSelect.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { FC, useRef, useState } from 'react';
+
+import { t, msg } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import {
+  Box,
+  Button,
+  Divider,
+  Menu,
+  MenuList,
+  Switch,
+  Typography,
+} from '@mui/material';
+import { ThemeConfig } from '@team-monite/sdk-demo/src/types';
+
+import { ThemeSelectMenuItem } from '@/components/ThemeSelect/ThemeSelectMenuItem';
+import { useThemeSelect } from '@/components/ThemeSelect/useThemeSelect';
+import { IconAngleDown, IconAngleUp, IconMoon, IconSun } from '@/icons';
+
+interface ThemeSelectorProps {
+  selectedTheme: ThemeConfig;
+  onThemeChange: (themeConfig: ThemeConfig) => void;
+}
+
+export const ThemeSelect: FC<ThemeSelectorProps> = (props) => {
+  const {
+    onThemeChange,
+    selectedTheme: { variant, mode },
+  } = props;
+
+  const menuButton = useRef<HTMLButtonElement | null>(null);
+
+  const { i18n } = useLingui();
+  const { modeName, themeName } = useThemeSelect(props);
+
+  const [open, setOpen] = useState(false);
+
+  const handleVariantChange = (selectedTheme: ThemeConfig) => {
+    onThemeChange(selectedTheme);
+  };
+
+  const variants: Record<ThemeConfig['variant'], string> = {
+    material: t(i18n)`Material UI`,
+    monite: t(i18n)`Monite`,
+  };
+
+  return (
+    <>
+      <Button
+        component="button"
+        className="theme-select"
+        ref={menuButton}
+        variant="outlined"
+        onClick={(event) => {
+          event.preventDefault();
+          setOpen(!open);
+        }}
+        sx={{ width: '100%', borderColor: 'neutral.80' }}
+      >
+        <Box
+          component="span"
+          display="flex"
+          flexDirection="row"
+          alignItems="center"
+          flex="1"
+          gap={1.5}
+        >
+          <Box component="span" display="flex" color="neutral.10">
+            {mode === 'dark' ? <IconMoon /> : <IconSun />}
+          </Box>
+          <Box component="span" display="flex" flexDirection="column" flex="1">
+            <Typography variant="body2" component="span" textAlign="left">
+              {themeName}
+            </Typography>
+            <Typography
+              className="theme-select__mode"
```

Comment:
As I have already advised, it is better to remove all `className` and use css-in-js features. We try not to rely on classes as they are harder to maintain in case of using ready-made component libraries.

Reply:
Clarified, let's keep the selector if it's needed.